framework/src/interfaces/PostprocessorInterface.C

```diff
@@ -20,36 +20,51 @@ PostprocessorInterface::PostprocessorInterface(const MooseObject * moose_object)
 }
 
 const PostprocessorValue &
-PostprocessorInterface::getPostprocessorValue(const std::string & name)
+PostprocessorInterface::getPostprocessorValue(const std::string & name, unsigned int index)
```

Comment:
These public interfaces should all have explicit checks or assertions on your vector access. I can see several potential pitfalls where accessing post processors in object constructors could result in seg faults here. These are public members.